README and Issue Templates (#101)
astronomer/cloud-docs: README.md

# Astronomer Cloud Documentation

> This repository contains all of the code and content for Astronomer Cloud's [documentation site](http://www.cloud-docs.astronomer.io). Documentation for other Astronomer product offerings is stored in the [`docs` repo](https://github.com/astronomer/docs).

Reviewer: ```suggestion
This repository contains all of the code and content that power Astronomer Cloud [documentation](http://www.cloud-docs.astronomer.io). Documentation for other Astronomer product offerings is stored in the [`docs` repo](https://github.com/astronomer/docs).
```

## Suggest a Change 

> Thank you for making a suggestion to our docs! Suggestions contributed in the form of GitHub issues are triaged and handled by the Astronomer team. 

Reviewer: ```suggestion
If you notice something in our documentation that is wrong, misleading, or could use additional context, the easiest way to make an impact is to create a GitHub issue in this repository. To do so,

1. Go to [Issues](https://github.com/astronomer/cloud-docs/issues)
2. Select **New Issue**
3. Depending on the change you have in mind, select a GitHub issue template.
4. Tell us what you think we can do better by answering the questions in the template.

GitHub issues are triaged by the Astronomer team and addressed promptly. Once you create a GitHub issue, our team may follow up with you with additional questions or comments. Once our team has addressed it, you'll get a notification via GitHub that the issue has been closed and that a change is now live.
```

> To make a suggestion, create a new issue in this repo. Please use the appropriate issue template and fill out all relevant template fields when submitting an issue. 

Reviewer: ```suggestion
```

## Contribute

> To contribute to Cloud docs, fork this repo and create a branch off of `main`. You can do this either directly in the GitHub repository or by clicking the "Edit this page" button on any published document.

Reviewer: ```suggestion
If you'd like to contribute to Astronomer Docs directly, you are welcome to create a Pull Request (PR) to this repository with your suggested changes. To do so:

1. Fork this repository
2. Create a branch off of `main`
3. Make your changes in that branch.
4. Submit a PR for review.
```

Once you have submitted a PR for your changes, Netlify will add a coment to your PR that includes a link to a staging website with your changes. 

> Small edits and typo fixes don't need to be linked to an issue and should be mergerd quickly. To get timely review on a larger contribution, we recommend first creating a detailed GitHub issue describing the problem and linking that within your PR. 

Reviewer: ```suggestion
Minor edits and typo fixes don't need to be linked to a corresponding GitHub issue and are expected to be reviewed and merged by the Astronomer team within 1-2 days. Larger contributions may take more time to validate but are very much appreciated. To get timely review on a significant set of changes, we recommend first creating a GitHub issue that describes the problem and is linked to the PR you've submitted.
```

> Every update to the main branch of this repository will trigger a rebuild of our production website at https://www.cloud-docs.astronomer.io. It might take a few moments for your merged changes to appear live on the website. 

Reviewer: ```suggestion
Every update to the `main` branch of this repository will trigger a rebuild of our production documentation page at https://www.docs.astronomer.io. It may take a few moments for your merged changes to appear.
```

> ### Build Locally with Docusaurus  

Reviewer: Should we be more prescriptive here? I've personally never had to build locally with Docusaurus - what instances specifically might the community need to interact with this?

I don't think we need to say `Docusaurus` here explicitly. I'd just leave it as `Build Astronomer Docs Locally` and then obviously have "Install Docusaurus" as a step.

> For larger contributions, such as new documents or screenshots, you likely need to interact with [Docusaurus](https://docusaurus.io/), which is our static site generator. Read the following sections for instructions on how to build and test your documentation changes locally. 

Reviewer: ```suggestion
If you want to submit a screenshot, GIF, or video to an existing doc OR add a new documentation file, you'll need to interact with [Docusaurus](https://docusaurus.io/), our static site generator. Read the following sections for instructions on how to build and test your documentation changes locally. 
```

#### Installation

Please read the [Docusaurus documentation](https://docusaurus.io/docs/installation#requirements) for information on installing the tools you'll need to work with Docusaurus locally. 

#### Local Development

To serve a local version of the Astronomer Cloud docs site with your changes, run:

```console

> yarn start

Reviewer: Could these be steps instead of H4s?

Author: While they do go in order, they're not necessarily a complete process. Someone could reference the Local Development section independent of the installation. Because of that, I think it makes sense to keep them as separate topics. Applied the same philosophy to the Develop Locally doc.

## Suggest a Change

If you notice something in our documentation that is wrong, misleading, or could use additional context, the easiest way to make an impact is to create a GitHub issue in this repository. To do so,

1. Go to [Issues](https://github.com/astronomer/cloud-docs/issues)
2. Select **New Issue**
3. Depending on the change you have in mind, select a GitHub issue template.
4. Tell us what you think we can do better by answering the questions in the template.

GitHub issues are triaged by the Astronomer team and addressed promptly. Once you create a GitHub issue, our team may follow up with you with additional questions or comments. Once our team has addressed it, you'll get a notification via GitHub that the issue has been closed and that a change is now live.

## Contribute

If you'd like to contribute to Astronomer Docs directly, you are welcome to create a Pull Request (PR) to this repository with your suggested changes. To do so:

1. Fork this repository
2. Create a branch off of `main`
3. Make your changes in that branch.
4. Submit a PR for review.

> Once you have submitted a PR for your changes, Netlify will add a coment to your PR that includes a link to a staging website with your changes.

Reviewer: ```suggestion
Once you have submitted a PR for your changes, Netlify will add a comment to your PR that includes a link to a staging website with your changes.
```

> Small edits and typo fixes don't need to be linked to an issue and should be mergerd quickly. To get timely review on a larger contribution, we recommend first creating a detailed GitHub issue describing the problem and linking that within your PR.

Reviewer: ```suggestion
Small edits and typo fixes don't need to be linked to an issue and should be mergerd quickly. To get a timely review on a larger contribution, we recommend first creating a detailed GitHub issue describing the problem and linking that within your PR.
```

Every update to the `main` branch of this repository will trigger a rebuild of our production documentation page at https://www.docs.astronomer.io. It might take a few moments for your merged changes to appear.

### Build Astronomer Docs Locally

> If you want to submit a screenshot, GIF, or a new documentation file, we recommend building and testing your documentation locally. Astronomer Cloud docs are built with [Docusaurus](https://docusaurus.io/), which is our static site generator. Read the following sections for instructions on how to build and test your documentation changes locally with Docusaurus.

Reviewer: ```suggestion
If you want to submit a screenshot, GIF, or a new documentation file, we recommend building and testing your documentation change locally. Astronomer Cloud docs are built with [Docusaurus](https://docusaurus.io/), which is our static site generator. Read the following sections for instructions on how to build and test your documentation changes locally with Docusaurus.
```
